tests/auto/tst_logins/tst_logins.qml

```diff
@@ -0,0 +1,68 @@
+/****************************************************************************
+**
+** Copyright (c) 2014 Jolla Ltd.
```

Comment:
Update

Reply:
Thanks; I've just pushed fixes to all of the copyright headers.